samples/cloudwatch/reference.md

```diff
@@ -0,0 +1,2926 @@
+# CloudWatch SAS Viya Metrics
+
+## By Dimensions
+This table lists the metrics associated with each set of dimensions.
+<details>
+  <summary>Click to expand</summary>
+<hr/>
+<details>
+  <summary>ClusterName,cas_node,cas_node_type,cas_server,job,namespace,pod (8)</summary>
+
+| Metric | Source |
+| ------ | ------ |
+| cas_node_fifteen_minute_cpu_load_avg | sas-cas |
+| cas_node_five_minute_cpu_load_avg | sas-cas |
+| cas_node_free_files | sas-cas |
+| cas_node_inodes_free | sas-cas |
+| cas_node_inodes_used | sas-cas |
+| cas_node_max_open_files | sas-cas |
+| cas_node_one_minute_cpu_load_avg | sas-cas |
+| cas_node_open_files | sas-cas |
+
+</details>
+<hr/>
+<details>
+  <summary>ClusterName,cas_node,cas_node_type,cas_server,connected,job,namespace,pod,uuid (1)</summary>
+
+| Metric | Source |
+| ------ | ------ |
+| cas_nodes | sas-cas |
+
+</details>
+<hr/>
+<details>
+  <summary>ClusterName,cas_server,job,namespace,pod (6)</summary>
+
+| Metric | Source |
+| ------ | ------ |
+| cas_grid_idle_seconds | sas-cas |
+| cas_grid_sessions_created_total | sas-cas |
+| cas_grid_sessions_current | sas-cas |
+| cas_grid_sessions_max | sas-cas |
+| cas_grid_start_time_seconds | sas-cas |
+| cas_grid_uptime_seconds_total | sas-cas |
+
+</details>
+<hr/>
+<details>
+  <summary>ClusterName,cas_server,job,namespace,pod,state (1)</summary>
+
+| Metric | Source |
+| ------ | ------ |
+| cas_grid_state | sas-cas |
+
```

Comment:
Are the source values (e.g. sas-cas, sas-go, etc.) meaningful outside of this document?  Do they correspond to specific pods or something?  If not why include the hypen?  And is the "sas" prefix needed/helpful?